moped-editor/src/views/dev/LookupsView/index.js

```diff
@@ -10,17 +10,17 @@ import Tooltip from "@mui/material/Tooltip";
 import Typography from "@mui/material/Typography";
 import IconButton from "@mui/material/IconButton";
 import Button from "@mui/material/Button";
-import LinkIcon from "@mui/icons-material/Link";
 import ArrowUpwardIcon from "@mui/icons-material/ArrowUpward";
 import Page from "src/components/Page";
 import RecordTable from "./RecordTable";
 import { TABLE_LOOKUPS_QUERY } from "src/queries/tableLookups";
 import { SETTINGS } from "./settings";
+import CopyLinkButton from "src/components/CopyLinkButton";
 
 /**
  * Converts a record key (e.g. moped_phases) into a URL hash, (e.g. #moped-phases)
  */
-const createRecordKeyHash = (recordKey) => `#${recordKey.replace("_", "-")}`;
+const createRecordKeyHash = (recordKey) => `#${recordKey.replaceAll("_", "-")}`;
```

Comment:
This small change makes our url hash for component tags look like `#moped-component-tags` instead of `#moped-componen_tags`. Not a big impact at all and mostly a fun reminder that `replaceAll` exists (for me at least 😅)